src/main.zig

```diff
@@ -609,10 +669,48 @@ const ServerCertificate = struct {
     bytes: []const u8,
     dn: []const u8,
     common_name: []const u8,
+    raw_subject_alternative_name: []const u8,
     public_key: x509.PublicKey,
     signature: asn1.BitString,
     signature_algorithm: SignatureAlgorithm,
     is_ca: bool,
+
+    const GeneralName = enum(u5) {
```

Comment:
This is a lousy name; I adopted it from the spec.

Reply:
The field names should be in snake_case